index.js

```diff
@@ -31,6 +33,40 @@ async function run() {
     }
     containerDef.image = imageURI;
 
+    if (environmentVariables) {
+
+      // If environment array is missing, create it
+      if (!Array.isArray(containerDef.environment)) {
+        containerDef.environment = [];
+      }
+
+      // Get pairs by splitting on newlines
+      environmentVariables.split('\n').forEach(function (line) {
+        // Trim whitespace and Split pairs on =
+        const pair = line.trim().split('=');
```

Comment:
I think it's possible that `line.trim()` results in the empty string `""`, for example in this scenario:
```
environment-variables: |
  NAME=value
```

Can we skip if it's empty?
```js
const trimmedLine = line.trim();
if (trimmedLine.length === 0) { return; }
const pair = trimmedLine.split('=');
```